package.json

```diff
@@ -1,15 +1,31 @@
 {
-  "name": "visual",
-  "dependencies": {
-    "powerbi-visuals-utils-dataviewutils": "1.2.0",
-    "lodash": "4.17.5"
-  },
-  "devDependencies": {
-    "@types/jquery": "2.0.41",
-    "@types/lodash": "4.14.55",
-    "jquery": "3.2.1",
-    "powerbi-visuals-tools": "1.13.1",
-    "powerbi-visuals-utils-testutils": "1.2.1",
-    "typescript": "2.3.4"
+    "name": "visual",
+    "description": "default_template_value",
+    "repository": {
+      "type": "default_template_value",
+      "url": "default_template_value"
+    },
+    "license": "MIT",
+    "scripts": {
+      "pbiviz": "pbiviz",
+      "start": "pbiviz start",
+      "package": "pbiviz package",
+      "lint": "tslint -c tslint.json -p tsconfig.json"
+    },
+    "dependencies": {
+      "@babel/runtime": "7.6.0",
+      "@babel/runtime-corejs2": "7.6.0",
+      "@types/d3": "5.7.2",
+      "core-js": "3.2.1",
+      "d3": "5.12.0",
+      "powerbi-visuals-api": "~4.7.0",
+      "powerbi-visuals-utils-dataviewutils": "2.2.1"
+    },
+    "devDependencies": {
+      "ts-loader": "6.1.0",
+      "tslint": "^5.18.0",
+      "tslint-microsoft-contrib": "^6.2.0",
```

Comment:
tslint is deprecated, please replaced it with [eslint-plugin-powerbi-visuals](https://www.npmjs.com/package/eslint-plugin-powerbi-visuals)

Reply:
done